e2e/json_util_test/test_load_from_url/test.yml

```diff
@@ -0,0 +1,9 @@
+query: >
+    CALL json_util.load_from_url("https://raw.githubusercontent.com/memgraph/mage/add-json-util/e2e/json_util_test/test_load_from_url/data.json") 
```

Comment:
Is there a possibility of sharing it on some MG cloud rather than using GitHub as storage

Reply:
Did that, next step is debugging the problem with opening that kind of file with the current request method. Will do asap :)

Reply:
The problem I had with making requests was the missing requests due to amazon external link I added https://download.memgraph.com/asset/mage/data.json. Then I figured that could happen with other kinds of links so I added header either way and now it works with raw github link, as well as the link that requires header.

Reply:
Nice!